Common/cpp/Tools/JSLogger.cpp

```diff
@@ -0,0 +1,23 @@
+#ifdef DEBUG
+
+#include "JSLogger.h"
+#include <memory>
+
+namespace reanimated {
+
+JSLogger::JSLogger(const std::shared_ptr<JSRuntimeHelper> &runtimeHelper)
+    : runtimeHelper_(runtimeHelper) {}
+
+void JSLogger::warnOnJS(const std::string &warning) const {
+  runtimeHelper_->scheduleOnJS(
+      [warning = warning, &runtimeHelper_ = runtimeHelper_]() {
```

Comment:
Will it work?
```suggestion
      [warning, &runtimeHelper = runtimeHelper_]() {
```

Reply:
▒▒▒▒▒▒▒▒▄▄▄▄▄▄▄▄▒▒▒▒▒▒▒▒
▒▒▒▒▒▄█▀▀░░░░░░▀▀█▄▒▒▒▒▒
▒▒▒▄█▀▄██▄░░░░░░░░▀█▄▒▒▒
▒▒█▀░▀░░▄▀░░░░▄▀▀▀▀░▀█▒▒
▒█▀░░░░███░░░░▄█▄░░░░▀█▒
▒█░░░░░░▀░░░░░▀█▀░░░░░█▒
▒█░░░░░░░░░░░░░░░░░░░░█▒
▒█░░██▄░░▀▀▀▀▄▄░░░░░░░█▒
▒▀█░█░█░░░▄▄▄▄▄░░░░░░█▀▒
▒▒▀█▀░▀▀▀▀░▄▄▄▀░░░░▄█▀▒▒
▒▒▒█░░░░░░▀█░░░░░▄█▀▒▒▒▒
▒▒▒█▄░░░░░▀█▄▄▄█▀▀▒▒▒▒▒▒
▒▒▒▒▀▀▀▀▀▀▀▒▒▒▒▒▒▒▒▒▒▒▒▒